shairport.c

```diff
@@ -2377,6 +2379,12 @@ int main(int argc, char **argv) {
         config.output_name == NULL ? "<unspecified>" : config.output_name);
   }
   config.output->init(argc - audio_arg, argv + audio_arg);
+  /* Also do an early open of the device to see it is correct */
```

Comment:
Can't do this -- we don't know what the device is being used for at power-up.

Reply:
hmm is it then just still a race condition on open when the first song is played?
Should there be some retry logic for that?
It was just really annoying to have to play a song to see if the config option for the sound setup is correct.
But I can just drop it here.